src/flycapture2.pyx

```diff
@@ -106,6 +113,8 @@ cdef class Context:
              "vendor_name": i.vendorName,
              "sensor_info": i.sensorInfo,
              "sensor_resolution": i.sensorResolution,
+             "driver_name" : i.driverName,  # Gige support
```

Comment:
I don't think you need those `# Gige support` comments all over. Are those just to mark your changes?